tests/Console/ConsoleScheduledEventTest.php

```diff
@@ -90,7 +70,7 @@ public function testTimeBetweenChecks()
         $app = m::mock('Illuminate\Foundation\Application[isDownForMaintenance,environment]');
         $app->shouldReceive('isDownForMaintenance')->andReturn(false);
         $app->shouldReceive('environment')->andReturn('production');
-        Carbon::setTestNow(Carbon::now()->startOfDay()->addHours(9));
+        $this->now->startOfDay()->addHours(9);
```

Comment:
Does this not change the context of the test? 

`Carbon::setTestNow()` sets the date that time currently is for all tests, so future tests such as `isBefore` or anything like that is relative to the date set on `setTestNow()`. Are these actually equivolent?


I see this as you getting the current time from `setUp` and then modifying the date, but that won't make make it work as it previously did for testing start times.